tempesta_fw/http_parser.c

```diff
@@ -1032,6 +1032,8 @@ __FSM_STATE(st_curr) {							\
 __FSM_STATE(RGen_HdrOtherN) {						\
 	__FSM_MATCH_MOVE(token, RGen_HdrOtherN, 0);			\
 	if (likely(*(p + __fsm_sz) == ':')) {				\
+		if (unlikely(!__fsm_sz))				\
+			TFW_PARSER_BLOCK(RGen_HdrOtherN);		\
```

Comment:
Please add the functional test case of sending h2c message to the current HTTP/1 parser.

Reply:
Unit tests was added into this PR and functional was added into tempesta-tech/tempesta-test#163